component/snackbar/snackbar.css

```diff
@@ -0,0 +1,25 @@
+.mantra-snackbar-success {
+    padding: 0.5rem;
+    background-color: var( --success-color);
+    width: max-content;
+    height: auto;
+    display: flex;
+    border-radius: 0.5rem;
+    font-size: 2rem;
```

Comment:
you could use your own declared variable here which is "--txt-size-nav" or could use utility class which you already have